miden/src/tools/mod.rs

```diff
@@ -34,32 +34,31 @@ impl Analyze {
         let stack_inputs = input_data.parse_stack_inputs()?;
         let advice_provider = input_data.parse_advice_provider()?;
 
-        let program_info: ProgramInfo = analyze(program.as_str(), stack_inputs, advice_provider)
-            .expect("Could not retrieve program info");
+        let program_info: ExecutionDetails =
```

Comment:
optional: maybe use `execution_details` as variable name to keep it consistent with the type?

Reply:
ofcourse! done :)